From ee0322c87b7f611757a6cdabe5b66f6dbf5117fe Mon Sep 17 00:00:00 2001
From: Arie Brown <abrown@habana.ai>
Date: Tue, 08 Sep 2020 13:35:42 +0300
Subject: [PATCH] [SW-19334] specs: file location changed

files moved from root directory to sub directory common in specs.
include lines changed accordingly to new location.

Change-Id: Ic7a7dea0e26f16b7ad674083ce64fadf93ba9c02
Signed-off-by: Arie Brown <abrown@habana.ai>
---

diff --git a/goya2_arc/common.c b/goya2_arc/common.c
index 8abebeb..eafccf5 100644
--- a/goya2_arc/common.c
+++ b/goya2_arc/common.c
@@ -18,7 +18,7 @@
 
 #include <hl_boot_if.h>
 #include <goya2/goya2_reg_map.h>
-#include <pci_ids.h>
+#include <common/pci_ids.h>
 #include <arcboot_export.h>
 
 #include <preboot.h>
diff --git a/goya2_arc_gnu/common.c b/goya2_arc_gnu/common.c
index a309275..1a3273e 100644
--- a/goya2_arc_gnu/common.c
+++ b/goya2_arc_gnu/common.c
@@ -22,7 +22,7 @@
 
 #include <hl_boot_if.h>
 #include <goya2/goya2_reg_map.h>
-#include <pci_ids.h>
+#include <common/pci_ids.h>
 #include <goya2_blocks_linux_driver.h>
 
 #include <preboot.h>
